src/plot_recipes.jl

```diff
@@ -59,10 +59,23 @@ argument. For additional options, consult the Plots.jl reference manual.
         indices = options.dict[:plot_indices]
     end
 
+    #=
     for i in indices
         vlist = hcat(vertices_list(sol.Xk[i])...).'
         @series (x, y) = vlist[:, 1], vlist[:, 2]
     end
+    =#
+
+    # Using single list and NaN separators as suggested in
+    vlist = Array{Array{Float64, 1}, 1}()
```

Comment:
@mforets: reference missing
Also, can we use `Vector` instead of `Array{X, 1}`?